vars/virtualenv.groovy

```diff
@@ -1,11 +1,21 @@
 import com.ableton.VirtualEnv
 
 
+/**
+ * Create a new {@link com.ableton.VirtualEnv} instance.
+ * @param python Python version or absolute path to Python executable.
+ * @param pyenvRoot Path of the pyenv installation.
+ * @return New {@link com.ableton.VirtualEnv} instance.
+ */
+VirtualEnv create(String python, String pyenvRoot) {
+  return VirtualEnv.create(script, python, pyenvRoot)
+}
+
 /**
  * Create a new {@link com.ableton.VirtualEnv} instance.
  * @param python Python version or absolute path to Python executable.
  * @return New {@link com.ableton.VirtualEnv} instance.
  */
 VirtualEnv create(String python) {
-  return VirtualEnv.create(this, python)
+  return VirtualEnv.create(python)
```

Comment:
Is this calling this https://github.com/AbletonDevTools/python-pipeline-utils/pull/56/commits/b0f1c3a783c3f162c1198b46d0c83a326f80a86f#diff-2424724bbebf6d79ee5c8cdb6edc3e0dR38?

Reply:
No, it's calling https://github.com/AbletonDevTools/python-pipeline-utils/commit/b0f1c3a783c3f162c1198b46d0c83a326f80a86f#diff-2424724bbebf6d79ee5c8cdb6edc3e0dR43. However this line is broken and needs to be reverted.

Reply:
Addressed in https://github.com/AbletonDevTools/python-pipeline-utils/pull/56/commits/5dde137bc6fa4b88dc716ebd3015019290443784.